cmd/reviewdog/main.go

```diff
@@ -28,6 +28,8 @@ import (
 	"github.com/mattn/go-shellwords"
 )
 
+var Export = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var Export should have comment or be unexported